engineyard/ey-cookbooks-dev-v7
# [FBZ-10450] Adds logentries support for ey-cookbooks-stable-v7

Description of your patch
-------------
Adds Logentries Support for `ey-cookbooks-stable-v7`

Recommended Release Notes
-------------
Adds Logentries Support for `ey-cookbooks-stable-v7` including installation of Logentries agent for the environment via environment variables.

Estimated risk
-------------
Low

Components involved
-------------
ey-base, ey-le

Dependencies
-------------
None

Description of testing done
-------------
* Setup `EY_LE_API_KEY` via environment variable, re-applied recipes on environment. 
* Checked if le was installed via `/etc/init.d/logentries status`
* Checked with InsightOps (Rapid7) , but it didn't workout saying as issue with Account ID from key received (Looked like clients were asked to migrate to the new portal after acquisition)

QA Instructions
-------------
* Receive a logentries API key for the old Logentries version (Not Rapid7 InsightOps)
* Check if `le register` succeeds.
* Check if logentries dashboard is pupulated with logs.
